common/lib/xmodule/xmodule/util/adaptive_learning.py

```diff
@@ -211,8 +219,12 @@ def get_knowledge_node_students(self, student_uid):
         """
         url = self.knowledge_node_students_url
         payload = {'student_id': student_uid, 'key_type': 'uid'}
-        response = requests.get(url, headers=self.request_headers, data=payload)
-        links = json.loads(response.content)
+        try:
+            links = []
+            response = requests.get(url, headers=self.request_headers, data=payload)
+            links = json.loads(response.content)
+        except requests.ConnectionError:
+            log.warning("Could not retrieve 'knowledge node student' objects for  user identified by student_uid {student_uid}.".format(student_uid=student_uid))
```

Comment:
@macornwell This line is too long; please add some line breaks to make it shorter.

Also, there's an extra space between "for" and "user" that we'll need to remove.